orders/orders/service.py

```diff
@@ -0,0 +1,64 @@
+from nameko.events import EventDispatcher
+from nameko.rpc import rpc
+from nameko_sqlalchemy import DatabaseSession
+
+from orders.models import DeclarativeBase, Order, OrderDetail
+from orders.schemas import OrderSchema
+
+
+class OrdersService:
+    name = 'orders'
+
+    db = DatabaseSession(DeclarativeBase)
+    order_schema = OrderSchema()
```

Comment:
It's probably cleaner to use `OrderSchema` directly, rather than attaching it as a class attribute.